cluster/common.sh

```diff
@@ -699,6 +714,11 @@ EOF
 KUBEPROXY_TEST_LOG_LEVEL: $(yaml-quote ${KUBEPROXY_TEST_LOG_LEVEL})
 EOF
     fi
+    if [ -n "${KUBEPROXY_FEATURE_CONFIG:-}" ]; then
+      cat >>$file <<EOF
+KUBEPROXY_FEATURE_CONFIG: $(yaml-quote ${KUBEPROXY_FEATURE_CONFIG})
```

Comment:
Why are you doing kube proxy? That's a node component.

Conversely, if this is intentional, you should do kubelet, too?